models/classes/TestCategoryPreset.php

```diff
@@ -172,7 +188,7 @@ public function toArray()
      * @return TestCategoryPreset the new instance
      * @throws common_exception_InconsistentData
      */
-    public static function fromArray(array $data)
+    public static function fromArray(array $data): TestCategoryPreset
```

Comment:
chore: totally optional and just sharing information. Since we have typehint + return type, there is no need for PHPDocs like `@param` or `@return`. It would also not break PHP Doc generation if you would remove it. We generally remove it as it is redundant and can make the code smaller. 

I good reason to keep is if you want to document these like `@param array $data Some info here`, but if not, you can remove safely